discovery/src/main/java/com/ibm/watson/developer_cloud/discovery/v1/model/NluEnrichmentEmotion.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2017 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.ibm.watson.developer_cloud.discovery.v1.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+import com.ibm.watson.developer_cloud.util.Validator;
+
+/**
+ * An object specifying the emotion detection enrichment and related parameters.
+ */
+public class NluEnrichmentEmotion extends GenericModel {
+
+  private Boolean document;
+  private List<String> targets;
+
+  /**
+   * Builder.
+   */
+  public static class Builder {
+    private Boolean document;
+    private List<String> targets;
+
+    private Builder(NluEnrichmentEmotion nluEnrichmentEmotion) {
+      document = nluEnrichmentEmotion.document;
+      targets = nluEnrichmentEmotion.targets;
+    }
+
+    /**
+     * Instantiates a new builder.
+     */
+    public Builder() {
+    }
+
+    /**
+     * Builds a NluEnrichmentEmotion.
+     *
+     * @return the nluEnrichmentEmotion
+     */
+    public NluEnrichmentEmotion build() {
+      return new NluEnrichmentEmotion(this);
+    }
+
+    /**
+     * Adds an targets to targets.
+     *
+     * @param targets the new targets
+     * @return the NluEnrichmentEmotion builder
+     */
+    public Builder addTargets(String targets) {
```

Comment:
We need `"x-item-name": "target"` on this array so that we get
```
public Builder addTarget(String target) {
```

We could probably make this change by hand here, rather than regenerate the whole SDK -- as long as we make sure to fix the swagger for future iterations.